core/dbt/parser/manifest.py

```diff
@@ -1232,6 +1233,24 @@ def _process_dimensions_and_relationships_for_metric(
                         )
                     metric.dimensions[to_model.name].append(col.name)
 
+        if (
+            not metric.time_grains
+            and not timestamp_col.time_grains
+            and "time_grains" not in metric.config._extra.keys()
```

Comment:
@dave-connors-3 is this all we needed for some config default of time_grains at the project/model level?

Reply:
Interesting - I agree keeping it in configs for now makes sense. What do you think about renaming it to `default_time_grains`? I think this might help differentiate that these are defaults applied to the project?

Reply:
fine by me!